schema.sql

```diff
@@ -8,3 +8,8 @@ CREATE TABLE animals (
     neutered boolean,
     weight_kg decimal
 );
+
+-- DAY 2
+alter table animals add column species varchar;
```

Comment:
- According to the project requirements, please limit the varchar to 255. Please fix this 🙏